scripts.js

```diff
@@ -65,6 +68,6 @@ function SetupUpdateEvents(curiosity) {
   const outputDiv = document.getElementById('output-display');
   
   const sliders = document.querySelectorAll('.slider');
-  sliders.forEach(slider => slider.addEventListener('change', updateAfterSliderChange));
```

Comment:
Curious as to why you changed this from 'change' to 'input'? is that a best practice?

Reply:
@ismaeldcom really someone gem here, i didn't know this. thank you.
really awesome to have you onboard.

Reply:
Oh I missed you reply here, Thanks for paying attention to the small things @ismaeldcom! 🙌